tests/benchmarks/garage/tf/algos/benchmark_trpo.py

```diff
@@ -218,59 +215,3 @@ def run_garage(env, seed, log_dir):
         dowel_logger.remove_all()
 
         return tabular_log_file
-
-
-def run_baselines(env, seed, log_dir):
```

Comment:
Same here

Reply:
It looks like the openai.baseline version is not support Tensorflow 2.0.